lightning/src/ln/peer_handler.rs

```diff
@@ -257,8 +257,13 @@ pub trait SocketDescriptor : cmp::Eq + hash::Hash + Clone {
 /// descriptor.
 #[derive(Clone)]
 pub struct PeerHandleError {
-	/// Used to indicate that we probably can't make any future connections to this peer, implying
-	/// we should go ahead and force-close any channels we have with it.
+	/// Used to indicate that we probably can't make any future connections to this peer (eg
+	/// because one of us or our peer had required features that the other was missing).
```

Comment:
nit: s/one of us or our peer/one of us

Reply:
Hmm, I prefer as-is - we use "we" in the same sentence to mean "this local LDK-based node" as well as mentioning "this peer". Saying "us" sounds like it could be a reference to the same thing as "we", so making it clear by saying "or our peer" feels helpful.